--- config.h	2025-06-03 08:10:26.427339839 -0600
+++ config.def.h	2025-06-03 22:09:47.979097975 -0600
@@ -11,9 +11,9 @@
     NULL; /* -p  option; prompt to the left of input field */
 static const char *colors[SchemeLast][2] = {
     /*     fg         bg       */
-    [SchemeNorm] = {"#ebdbb2", "#282828"},
-    [SchemeSel] = {"#ebdbb2", "#458588"},
-    [SchemeOut] = {"#ebdbb2", "#8ec07c"},
+    [SchemeNorm] = {"#d3c6aa", "#2f383e"},
+    [SchemeSel]  = {"#d3c6aa", "#a35e4e"},
+    [SchemeOut]  = {"#2f383e", "#a7c080"},
 };
 /* -l option; if nonzero, dmenu uses vertical list with given number of lines */
 static unsigned int lines = 0;
